conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
+# All Rights Reserved
+#
+# conda is distributed under the terms of the BSD 3-clause license.
+# Consult LICENSE.txt or http://opensource.org/licenses/BSD-3-Clause.
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from .core import repodata as _impl
+from .connection import CondaSession
+from os.path import join
+
+
+class RepoData(object):
+    """This object represents all the package metainfo of a single channel."""
+
+    _data = {}
```

Comment:
Hmm, but this isn't a cache at all. It's the data. (Unless you think of the RepoData object as a proxy that really refers to the in-channel data, so all the local data would be the cache.)
I don't have a strong opinion about the naming otherwise, though would tend to use a single underscore for private members (I don't mind leading or trailing, as long as it's consistent).